app/lib/fsHelper.js

```diff
@@ -18,7 +28,7 @@ function saveJsonSync(obj, filename) {
 
 function loadJsonSync(filename) {
   const path = `${OUTPUT_DIR}/${filename}.json`;
-  const jsonString = fs.existsSync(path) ? fs.readFileSync(path, 'utf8') : undefined;
+  const jsonString = fileExists(path) ? fs.readFileSync(path, 'utf8') : undefined;
```

Comment:
I think for now we should move it back to existsSync.

It would appear that `exists` was deprecated because of the order of parameters.
The suggested replacement methods throw exceptions rather than return a status.
As you are already catching an exception the suggestion of trying the action that might fail and catching that exception makes sense.